src/main/kotlin/no/nav/syfo/bootstrap/HttpClients.kt

```diff
@@ -39,41 +40,45 @@ class HttpClients(environment: Environment) {
                 }
             }
         }
-        expectSuccess = false
-    }
-    private val retryConfig: HttpClientConfig<CIOEngineConfig>.() -> Unit = {
-        config().apply {
-            install(HttpRequestRetry) {
-                maxRetries = 3
-                delayMillis { retry ->
-                    retry * 500L
+        install(HttpRequestRetry) {
+            constantDelay(100, 0, false)
+            retryOnExceptionIf(3) { _, throwable ->
+                log.warn("Caught exception ${throwable.message}")
```

Comment:
går det an å si hvilket kall vi retryer?

Reply:
Det skal gå ann, kan legge det til: 
`   log.warn("Retrying for statuscode ${response.status.value}, for url ${request.url}")`